### PR TITLE
Reset release notes and updated host version to 4.3.0

### PR DESCRIPTION
**Pull request by v-vishalr**
### Issue describing the changes in this PR
Resetting release notes and setting new version after merge to release branch for sprint 118 release.

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
+
 ### Release notes
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.0.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.0.0)
```
Comment (VpOfEngineering) [inline release_notes.md:5]: @v-vishalr , can you keep just this line? This change didn't make it into `v4.2.0`.

Comment (vrdmr) [inline release_notes.md:5]: This is incorrect - my changes didn't go into 4.2.0 - Could you please validate?

Comment (vrdmr) [reply release_notes.md:5]: My changes went in 6 hours ago - https://github.com/Azure/azure-functions-host/pull/8156

**Review by vrdmr — DISMISSED** [review]

**Review by v-vishalr — COMMENTED** [review]
Added python worker version line

File: release_notes.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Afeature+is%3Aclosed) ]
-- Update PowerShell Worker 7.0 to 4.0.1570 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.1570)
-- Update PowerShell Worker 7.2 to 4.0.1567 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.1567)
-- Update App Service Authentication/Authorization on Linux Consumption from 1.4.10 to 1.4.13 [Release Note](https://github.com/Azure/app-service-announcements/issues/354)
+**Release sprint:** Sprint 118
+[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+118%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+118%22+label%3Afeature+is%3Aclosed) ]
+
+Update Python Worker Version to [4.0.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.0.0)
```
Comment (VpOfEngineering) [inline release_notes.md]: Please move this to be between line 7 and line 5 - thank you!
```suggestion
Update Python Worker Version to [4.0.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.0.0)
```

**Review by v-vishalr — COMMENTED** [review]
added changes

**Comment by VpOfEngineering** [comment]
@v-vishalr this is what we are requesting the `release_notes.md` file to look like. Can you please update it to match what is below?

```
### Release notes
- Update Python Worker Version to [4.0.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.0.0)

**Release sprint:** Sprint 118
[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+118%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+118%22+label%3Afeature+is%3Aclosed) ]
```

**Review by v-vishalr — COMMENTED** [review]
changes to release note

**Review by VpOfEngineering — DISMISSED** [review]

**Comment by VpOfEngineering** [comment]
LGTM, @soninaren ?

**Review by soninaren — DISMISSED** [review]

**Comment by VpOfEngineering** [comment]
Apologies, meant to tag @vrdmr

File: release_notes.md (hunk 1 above)
Comment (vrdmr) [reply release_notes.md:5]: @VpOfEngineering - When looking at the [release/4.x](https://github.com/Azure/azure-functions-host/tree/release/4.x) branch, I do see my commit in there. Checking again if it did go into v4.2.0 or not?

I personally feel we should create tags sooner, before OGF - or at least some way to confirm what is getting released. I don't have a clear idea.

Comment (vrdmr) [reply release_notes.md:5]: @v-vishalr Could you please help answer the above?

Comment (VpOfEngineering) [reply release_notes.md:5]: It is included, I see that now. @v-vishalr please remove the python line and add 
```
### Release notes
<!-- Please add your release notes in the following format:
- My change description (#PR)
 -->
```

**Review by v-vishalr — COMMENTED** [review]
updates to release notes

**Review by VpOfEngineering — APPROVED** [review]
LGTM

**Review by vrdmr — APPROVED** [review]
🚢

**Pull request merged**